### PR TITLE
Blank interactive template

### PR DESCRIPTION
**Pull request by phamann**
In partnership with the interactive team, we are experimenting ways of hiding the global page furniture for "special" interactives. This is the first test. 
- Look for interactives which have bootscripts in a specific location
- Set hideUi boolean in content model
- Hide all non-essential fragments within the main template

The next step with be to model the `hideUi` booelan in Composer interface and CAPI representation.

/cc @gklopper @andymason @daanlouter

### CODE REVIEW

**Comment by phamann** [comment]
Diff is best viewed with the [`?w=1`](https://github.com/guardian/frontend/pull/8247/files?w=1) hack

File: common/app/conf/switches.scala [hunk 1]
```diff
@@ -364,6 +364,11 @@ object Switches {
     sellByDate = never
   )
 
+  val HideInteractiveUi = Switch("Feature", "hide-interactive-ui",
+    "If this is switched on interactives can be rendered without page funiture.",
```
Comment (gklopper) [inline common/app/conf/switches.scala]: spelling - I'm soooooooo disappointed

File: common/app/model/content.scala [hunk 2]
```diff
@@ -738,6 +738,9 @@ class Interactive(content: ApiContentWithMeta) extends Content(content) {
   override lazy val analyticsName = s"GFE:$section:$contentType:${id.substring(id.lastIndexOf("/") + 1)}"
 
   override lazy val metaData: Map[String, JsValue] = super.metaData + ("contentType" -> JsString(contentType))
+  lazy val hideUi: Boolean = body.map{ b =>
+      Jsoup.parseBodyFragment(b).body().getElementsByClass("element-interactive").attr("data-interactive").contains("/visuals/")
+    }.getOrElse(false)
 }
```
Comment (gklopper) [inline common/app/model/content.scala]: `body.exists(...)`

**Comment by gklopper** [comment]
Just the `hideUi` thing we spoke about then :+1:

File: common/app/conf/switches.scala [hunk 3]
```diff
@@ -364,6 +364,11 @@ object Switches {
...
+    safeState = On, sellByDate = new LocalDate(2015, 2, 28)
```
Comment (gklopper) [inline common/app/conf/switches.scala]: 28 Feb is a Saturday...

**Pull request merged**